common/src/View/MapDocument.h

```diff
@@ -96,7 +96,7 @@ namespace TrenchBroom {
             std::unique_ptr<MapViewConfig> m_mapViewConfig;
             std::unique_ptr<Grid> m_grid;
 
-            using ActionList = std::list<Action>;
+            using ActionList = std::vector<std::unique_ptr<Action>>;
```

Comment:
I know it was me who introduced all of these, but I'm starting to think that we should get rid of these type aliases. I think I'd just replace this with the actual type.

Reply:
I am on board with that :) there are probably a few really long ones that make sense to keep